.buildkite/pipeline.yml

```diff
@@ -30,6 +31,42 @@ steps:
           - Bugsnag-with-android-64bit.unitypackage
         upload:
           - test/desktop/maze_output/*
+    commands:
+      - cd test/desktop
+      - ./features/scripts/build_maze_runner.sh
+    artifact_paths:
+      - test/desktop/features/fixtures/Mazerunner-2017.4.40f1.app.zip
+
+  - label: Build Unity 2018 desktop test fixture
+    key: 'cocoa-2018-fixture'
+    depends_on: 'build-artifacts'
+    timeout_in_minutes: 30
+    agents:
+      queue: opensource-mac-unity
+    env:
+      UNITY_VERSION: "2018.4.34f1"
+    plugins:
+      artifacts#v1.2.0:
+        download:
+          - Bugsnag.unitypackage
+          - Bugsnag-with-android-64bit.unitypackage
+    commands:
+      - cd test/desktop
+      - ./features/scripts/build_maze_runner.sh
+    artifact_paths:
+      - test/desktop/features/fixtures/Mazerunner-2018.4.34f1.app.zip
+
+  - label: Run e2e tests for Unity 2017.4.40f1
+    depends_on: 'cocoa-2017-fixture'
+    timeout_in_minutes: 30
+    agents:
+      queue: opensource-mac-cocoa
```

Comment:
Ah, looks like this is now classed as legacy and newer servers are explicitly providing the macOS version (e.g. `opensource-mac-cocoa-10.15`).  Leave it as-is, though, and I can adjust on the next change (or we can just change the agent config if that's what we want).